commands/mentor.py

```diff
@@ -0,0 +1,28 @@
+import requests
+import random
+from disco.bot import Bot,Plugin
+from disco.types.message import MessageEmbed
+from config_mock import AnnounceBotConfig, TestPluginConfig
+
+MENTOR_ID = 487807730506006540
+
+@Plugin.with_config(TestPluginConfig)
+class MentorPlugin(Plugin):
+
+    def get_avail_mentors(self):
+       return [u.user.id for u in self.bot.client.state.guilds.get(487611837664198679).members.values() if MENTOR_ID in u.roles and u.user.presence.status != "OFFLINE"]
```

Comment:
not sure we wana consider people who are afk or even DND for pinging? maybe first try online and only then those, afk can be around but not always, and DND won't even be notified, not sure what the best option would be

Reply:
It should be `u.user.presence.status = "ONLINE"`

Reply:
sounds good to me, changed